config/src/config/state_sync_config.rs

```diff
@@ -126,8 +126,8 @@ impl Default for StorageServiceConfig {
             max_network_chunk_bytes: MAX_MESSAGE_SIZE as u64,
             max_state_chunk_size: 4000,
             max_subscription_period_ms: 5000,
-            max_transaction_chunk_size: 5000,
-            max_transaction_output_chunk_size: 5000,
+            max_transaction_chunk_size: 3000,
```

Comment:
@igor-aptos, do we want to do this as well as changing to execution mode? I worry that changing both things at once might cause noise https://github.com/aptos-labs/aptos-core/pull/7427